src/app/shared/dso-page/dso-edit-menu/dso-edit-menu-section/dso-edit-menu-section.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="!canActivate" class="dso-button-menu mb-1"
+     [title]="itemModel.text | translate"
+     [ngbTooltip]="itemModel.text | translate">
+    <a *ngIf="!section.model.disabled" class="d-flex flex-row flex-nowrap"
+       [routerLink]="itemModel.link"
+       href="javascript:void(0);">
+        <button class="btn btn-dark btn-sm" [disabled]="section.model.disabled">
```

Comment:
Same here, it'd be good to add an `aria-label` or `title` to this button for accessibility.